### DISCUSSION TITLE
Why can't I render PDF?

### DISCUSSION THREAD

**Discussion by rookie-orange** [TaTo30/vue-pdf#146] [Q&A]
I created an empty project to try out this library("@tato30/vue-pdf": "^1.11.0", "vue": "^3.4.37").
But, Why can't I render PDF?The page contains only blank HTML tags.
![QQ_1725524908690](https://github.com/user-attachments/assets/a7cc5860-7ffd-42e1-9bcb-21895f3ec2f0)
![QQ_1725524952444](https://github.com/user-attachments/assets/8a558724-a517-4b3f-a0e1-1810171d316e)
I hope you can help me solve the problem. Please.

**Comment by TaTo30** [comment]
Version `1.11.0` had some issues with a version mismatch between pdf.js worker and pdf.js module (see #142).

Update to `1.11.1` and try again
